### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by oschrenk** [spray/spray-json#346]
Release notes from JWT scala ( https://github.com/jwt-scala/jwt-scala/releases/tag/v9.0.3 )  imply that spray-json is no longer maintained.

I thought it was feature complete and there was just no change anymore. Any reason to believe different?

**Comment by jrudolph** [comment]
Indeed, not much has changed in the status of the project. Right now, no new changes are planned but we will try to accommodate for the most pressing issues when needed.

**Comment by raboof** [comment]
Agreed. The way I look at it is:

It is maintained in the sense that 'important things' happen: 1.3.6 has been [released for Scala 3](https://github.com/spray/spray-json/pull/345) and I'm confident any security issues would be dealt with swiftly.

"feature complete" is in the eye of the beholder: spray-json is certainly a very usable json library as it is now, but there's always something to be desired. There's a bunch of PR's open, some of which are arguably-potentially-interesting, and it's true no-one is currently putting in energy to review and merge those. Speaking for myself, realistically I'd rather have spray-json as it is now, compared to having it with possible improvements, but running the risk of breaking things for existing users and having to scramble to rectify that situation later...
